models/clm/tutorial/README.rst

```diff
@@ -0,0 +1,1591 @@
+the error and resolve the issue.  Contact dart@ucar.edu if necessary.
+
+
+Just because an assimilation ran successfully (without errors) does not mean
+it ran with good performance. A simple, first check after any assimilation is to 
+make sure:
+
+1) Observations have been  accepted
+2) The CLM posterior member values are updated from their prior values
+
+A quick way to confirm observation acceptance and the posteriors have been updated
+is through the ``obs_seq.final`` file located in your case run folder. Below we provide
+an example of a successful update (``clm_obs_seq.2011-01-02-00000.final``)  which is
+derived from the same leaf area observation in ``obs_seq.2011-01-02-00000`` as described
+in Step 7.  
+
+::
+
+ OBS            3
+   6.00864688253571
+   5.44649167346675
+   5.45489142211957
+   5.45808308572015
+   3.479253215076174E-002
+   3.469211455745640E-002
+   5.41512442472872
+   5.41843086313655
+   5.44649167346675
+   5.44970758027390
+   5.50877061923831
+   5.51180677764943
+   5.46367193547511
+   5.46683825691274
+   5.44039845768897
+   5.44363195062815
+   0.000000000000000E+000
+   0.000000000000000E+000
+           2           4          -1
+ obdef
+ loc3d
+     5.235987755982989         0.000000000000000        -888888.0000000000     -2
+ kind
+          23
+     0     149750
+  0.200000000000000
+
+::
+
+   <observation sequence number>
+   <observation value>
+   <true observation value>
+   <prior ensemble mean>
+   <posterior ensemble mean>
+   <prior ensemble spread>
+   <posterior ensemble spread>
+   <prior member 1>
+   <posterior member 1>
+   <prior member 2>
+   <posterior member 2>
+   <prior member 3>
+   <posterior member 3>
+   <prior member 4>
+   <posterior member 4>
+   <prior member 5>
+   <posterior member 5>
+   <data product QC>
+   <DART quality control>
+ 
+ obdef
+ loc3d
+     <longitude>    <latitude>   <vertical level>   <vertical code>
+ kind
+    <observation quantity number>
+    <seconds>     <days>
+    <observation error variance>
+
+
+In the example above, the observation has been **accepted**
+denoted by a ``DART quality control value = 0``. If the 
+``DART quality control value =7`` this indicates the observation has
+fallen outside the ``outlier_threshold`` value and is rejected. 
+For more details on the DART quality control variables read the  
+`documentation. <https://docs.dart.ucar.edu/en/latest/assimilation_code/modules/assimilation/quality_control_mod.html>`__
+
+First, in your own tutorial assimilation confirm that this observation
+(and other observations) was accepted.
+
+Second, confirm that the ``posterior member`` values have been updated
+from their respective ``prior member`` values.  In general the
+``posterior member`` and ``posterior ensemble mean`` values should become
+closer to the `observation value` as compared to the prior values.
+
+
+::
+
+ > cd <caseroot>
+ > vi clm_obs_seq.2011-01-02-00000.final
```

Comment:
```suggestion
  cd <caseroot>
  less clm_obs_seq.2011-01-02-00000.final
```

You could suggest less (or cat if there are not many observations) here since this is just viewing the output.